internal/provider/role_resource_test.go

```diff
@@ -29,25 +39,25 @@ func TestAccRoleResource(t *testing.T) {
 				Config: createFileConfig,
 				Check: resource.ComposeAggregateTestCheckFunc(
 					// Verify Trust Provider Name
-					resource.TestCheckResourceAttr(resourceID, "name", createResourceName),
+					resource.TestCheckResourceAttr(resourceID, "name", newName),
 					// Verify dynamic values have any value set in the state.
 					resource.TestCheckResourceAttrSet(resourceID, "id"),
 					// Verify placeholder ID is set
 					resource.TestCheckResourceAttrSet(resourceID, "id"),
 				),
 			},
+			// Test Aembit API Removal causes re-create with non-empty plan
+			{Config: string(createFile), Check: testDeleteRole(resourceID), ExpectNonEmptyPlan: true},
```

Comment:
Do we not want to use the same createFileConfig variable here?

Reply:
Good catch - fixed